data/merchant intro.txt

```diff
@@ -0,0 +1,138 @@
+event "Merchant Start"
+	date 4 1 3016
+
+mission "Merchant Start 1"
+	name "Shipment to <destination>"
+	description "Take <cargo> to <destination>"
+	cargo "Medieval Weaponry" 10
+	to offer
+		"assisted free worlds" < 3
+		has "event: Merchant Start"
+	source
+		government "Republic" "Syndicate" "Free Worlds" "Neutral"
+	destination
+		distance 5 8
+		government "Republic" "Free Worlds" "Syndicate" "Neutral"
+	to fail
+		has "chosen sides"
+	on offer
+		conversation
+			`As you enter the bustling spaceport, a man wearing a dark blue suit walks up to you. "Excuse me, would you mind ferrying these <cargo> to <destination>? Our original contact bailed out on us after they were attacked days ago while transporting the cargo."`
+			choice
+				`	"Of course."`
+					accept
+				`	"No thanks."`
+					decline
+	on complete
+		payment 30000
+		conversation
+			`Men quickly unload the cargo from your hold. A woman in a dark blue suit, which is almost the same suit as the man you met for this job, thanks you. "We have another mission that requires a captain to transport it. If you would like to take that offer, meet me in the spaceport after this."`
+
+mission "Merchant Start 2"
+	name "Passenger transport"
+	description "Help out the merchants by bringing a family to <destination> by <date>"
+	passengers 3
+	deadline 7
+	to offer
+		has "Merchant Start 1: done"
+	source
+		government "Republic" "Syndicate" "Free Worlds" "Neutral"
+	destination
+		distance 5 5
+		government "Republic" "Free Worlds" "Syndicate" "Neutral"
+	on offer
+		conversation
+			`You walk into the spaceport, looking for the lady. You quickly find her striking dark blue suit in the crowd, clearly standing out from everyone's frivolous apparel as you walk over to her. "What's the job you're offering me?" She smiles, "Well, this family desperately needs transport to <destination> by <date>," she says, pointing out a group of people at a corner of the spaceport, "they're in a bit of troublesome economic crisis at the moment"`
+			choice
+				`	"I'll do it."`
+					accept
+				`	"I don't have the time."`
+					decline
+	on complete
+		payment 40000
+		conversation
+			`The family thanks you profusely, grateful for meeting such a generous captain. You watch the family walk off and exchange greetings with another man in a blue suit, the blue-suited man walks up to you and says, "Meet me in the spaceport, don't dally."`
```

Comment:
How about:
"You watch the family walk off and greet another man in a blue suit. You're about to turn back to your ship when yet another man in a blue suit approaches you. "Meet me in the spaceport," he says seriously. "Don't dally."

Reply:
Hmm. There's only one man with a blue suit, he just greets the family then walks up to you.
The "he says seriously" might work, yet, it's a bit bulky

Reply:
I only made separated the quotes because there needs to be a full stop in between there anyways. So if you want it in 1 quote, it should be "Meet me in the spaceport; don't dally." or "Meet me in the spaceport. Don't dally." But I personally like it with the "he says seriously" separating the two. Gives it an air of authority.

Reply:
How about "he says, in an official tone"?
Or "he says sternly"
I'm not sure why, but I just don't like the word "seriously" in this context